tests/fixtures.py

```diff
@@ -206,7 +206,7 @@ def jobtask_mock(auth_mock):
             f"{auth_mock._endpoint()}/projects/{auth_mock.project_id}/jobs/{job_id}"
             f"/tasks/"
         )
-        m.get(url=url_jobtask_info, json={"data": {"xyz": 789}, "error": {}})
+        m.get(url=url_jobtask_info, json={"data": [{"xyz": 789}], "error": {}})
```

Comment:
Is data now a list? Did this change in the API?

Reply:
Yes jobtasks data is always a list of dictionaries. mock test wasn't correct, added a list assertion now.